src/resultwizard/__init__.py

```diff
@@ -1,3 +1,3 @@
-from api.config import config_init, config
+from api.config import config
```

Comment:
Nice, that's a good change since this `config_init` and `config` was indeed a bit weird to use in the first place.